packages/editor/package.json

```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@theia/core": "^0.3.17",
     "@theia/languages": "^0.3.17",
+    "@theia/filesystem": "^0.3.17",
```

Comment:
so editor cannot be used without filesystem anymore?

Reply:
Should I not be introducing a dependency to `filesystem` in order to fix the issue ?
I had to determine if a URI truly exists since we are currently displaying and referringing to locations which no longer exist. I could fix the issue from the `quick-open-file` but thought that if I update the service, other consumers of the service can benefit from the same change.

Reply:
@vince-fugnitto, I can fully understand the fix you made but I prefer living with this defect instead of introducing the `filesystem` in `editor`.

https://github.com/theia-ide/theia/blob/63365dc1d787df19f47898a2aa0204bb4c2dadda/packages/editor/src/browser/editor-navigation-contribution.ts#L57

Perhaps, later we will have the silver-bullet solution for this issue. BTW, did you see [this](https://github.com/theia-ide/theia/pull/1649#issuecomment-379671829) comment from Anton?

Reply:
I found this too: https://github.com/theia-ide/theia/issues/1671

Reply:
Thanks @kittaakos for clarifying, I can fallback to my original solution before I found the root cause and that is to add handling when attempting to open a file from the quick-open which no longer exists and in that case inform the users through the message-service